rubi/rubi/client.py

```diff
@@ -689,6 +689,37 @@ def get_offers(
         )
         return df
 
+    def get_trades(
+        self,
+        taker: Optional[str] = None,
+        from_address: Optional[str] = None,
+        pair_name: Optional[str] = None,
+        book_side: Optional[OrderSide] = OrderSide.NEUTRAL,
+        take_gem: Optional[str] = None,
```

Comment:
On the comment I on the `data/market.py` I think that this method is the one we abstract for the user. As such this one should have the `pair_name` but not the `take_gem` and `buy_gem` params.

Reply:
fo sho